mysql-test/suite/tokudb/t/cluster_key_part.test

```diff
@@ -5,19 +5,25 @@ source include/have_partition.inc;
 set default_storage_engine='tokudb';
```

Comment:
Not a bad idea and another point where TokuDB might have some mtr weakness. Scopewise, I don't really care, if we can make a better test when we touch a feature, we should and automatically assume it is within the scope, especially with a feature like this where we are opening up an entire new area (well, new to PS) that hasn't been executed at all due to lifting this limitation.

I believe there is a cluster_key_something test that I can snip a few lines from and append to this test to validate that it is functioning beneath the partition handler.